src/ssz/type/byte_list.zig

```diff
@@ -147,6 +147,7 @@ pub fn ByteListType(comptime _limit: comptime_int) type {
                 }
 
                 const nodes = try allocator.alloc(Node.Id, chunk_count);
+                defer allocator.free(nodes);
```

Comment:
is this part of this PR? the fix is correct but we don't apply TreeView for BeaconState yet

Reply:
nvm, I saw the `merkle_proof `test